app/templates/common/_footer.html

```diff
@@ -1,9 +1,11 @@
-<footer class="footer">
-    <div class="fixed-bottom">
- <div class="container-fluid">
-  <div class="row text-center">
-   <div class="col-lg-12">Copyright © 2021 Soengram Adapted from <a href="instagram.com">Instagram</a></div>
-        </div>
+<footer class="footer text-center col-lg-12">
+    <div class="container-fluid row text-center col-lg-12">
+        <div class="container-fluid">
+            <div class="row text-center">
+                <div class="col-lg-12">
+                    Copyright © 2021 Soengram adapted from <a href="instagram.com">Instagram</a> and customized by Soengram team
```

Comment:
href redirects to a non existent URL on Soengram instead of Instagram itself. The entire URL is required `https://www.instagram.com/`

Reply:
- modified Instagram link with complete URL to Instagram